src/nginx/habitat/config/resolver.lua

```diff
@@ -0,0 +1,53 @@
+module("resolver", package.seeall)
+local upstream_resolver = { }
```

Comment:
We can defer it to post-merge, but I really want to understand how much of this routing we really need to do now that the chef, erchef, an account services are all in erchef.